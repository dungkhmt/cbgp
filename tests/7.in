30 30
30 50
17 10
22 0
5 0
25 13
8 29
16 22
19 0
10 6
15 0
20 3
27 7
11 15
10 12
1 3
16 2
22 3
4 5
19 24
17 1
26 2
17 27
23 0
19 25
22 8
29 14
19 9
21 22
16 18
28 0
12 9
5 2
19 4
13 0
13 4
22 1
23 2
24 2
19 13
19 6
23 15
14 8
22 18
26 16
21 8
19 2
28 19
12 1
16 15
12 14
22 20
1
15 15 35 5 18 5 34 6 14 7 16 7 46 0 43 2 35 2 15 1 45 1 22 7 40 1 12 9 37 2 21 6 24 3 38 8 23 2 3 0 33 7 34 3 28 2 19 6 15 6 12 8 18 4 41 2 0 0 36 3 