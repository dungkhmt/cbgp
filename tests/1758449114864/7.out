(16, 9), (21, -1), (4, -1), (24, 12), (7, 28), (15, 21), (18, -1), (9, 5), (14, -1), (19, 2), (26, 6), (10, 14), (9, 11), (0, 2), (15, 1), (21, 2), (3, 4), (18, 23), (16, 0), (25, 1), (16, 26), (22, -1), (18, 24), (21, 7), (28, 13), (18, 8), (20, 21), (15, 17), (27, -1), (11, 8), (4, 1), (18, 3), (12, -1), (12, 3), (21, 0), (22, 1), (23, 1), (18, 12), (18, 5), (22, 14), (13, 7), (21, 17), (25, 15), (20, 7), (18, 1), (27, 18), (11, 0), (15, 14), (11, 13), (21, 19), 
numberIntersection=2 
minAngle=0.1651486774146269 
sumAngle=41.48815488594547 
minEdgeLength=1.0 
sumEdgeLength=1096.9582342067529 
minDistance=0.2554798301116468 
sumDistance=45570.94360125578 

-2.0000000000 0.1651486774 41.4881548859 1.0000000000 1096.9582342068 0.2554798301 45570.9436012558
0: (23, 31),
1: (38, 8),
2: (22, 35),
3: (57, 21),
4: (51, 17),
5: (4, 3),
6: (120, 0),
7: (83, 18),
8: (1, 37),
9: (3, 3),
10: (37, 41),
11: (0, 38),
12: (65, 7),
13: (0, 86),
14: (43, 18),
15: (21, 20),
16: (3, 2),
17: (30, 28),
18: (37, 2),
19: (29, 40),
20: (58, 26),
21: (37, 42),
22: (40, 12),
23: (37, 6),
24: (51, 7),
25: (30, 16),
26: (62, 1),
27: (120, 1),
28: (34, 74),
29: (120, 120),
23 31 38 8 22 35 57 21 51 17 4 3 120 0 83 18 1 37 3 3 37 41 0 38 65 7 0 86 43 18 21 20 3 2 30 28 37 2 29 40 58 26 37 42 40 12 37 6 51 7 30 16 62 1 120 1 34 74 120 120 
0: (23, 31), 1: (38, 8), 2: (22, 35), 3: (57, 21), 4: (51, 17), 5: (4, 3), 6: (120, 0), 7: (83, 18), 8: (1, 37), 9: (3, 3), 10: (37, 41), 11: (0, 38), 12: (65, 7), 13: (0, 86), 14: (43, 18), 15: (21, 20), 16: (3, 2), 17: (30, 28), 18: (37, 2), 19: (29, 40), 20: (58, 26), 21: (37, 42), 22: (40, 12), 23: (37, 6), 24: (51, 7), 25: (30, 16), 26: (62, 1), 27: (120, 1), 28: (34, 74), 29: (120, 120), 
numberIntersection=2 minAngle=0.1651486774146269 sumAngle=41.48815488596049 minEdgeLength=1.0 sumEdgeLength=1096.958234206752 minDistance=0.2554798301116468 sumDistance=45570.94360125586 
Time taken: 11.346 s
